src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -278,6 +278,12 @@ private void StartLogging()
 
             _logger.Log(PsesLogLevel.Diagnostic, "Logging started");
         }
+        
+        private void SetDistributionChannel()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'SetDistributionChannel'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4819096)